### PR TITLE
Device Provisioning Service

### PR DESCRIPTION
**Pull request by montgomp**
## Description

Re-opening device provisioning service PR due to swagger name changes.  https://github.com/Azure/azure-rest-api-specs/tree/master/specification/deviceprovisioningservices/resource-manager
---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by montgomp** [comment]
@shahabhijeet is it possible to get someone assigned to review this PR? The initial PR was opened mid December and closed due to requested changes to Swagger naming.

File: src/SDKs/DeviceProvisioningServices/Management.DeviceProvisioningServices/Microsoft.Azure.Management.DeviceProvisioningServices.csproj [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+	
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.DeviceProvisioningServices</PackageId>
+    <Description>Provides management capabilities for Microsoft Azure IoT Device Provisioning Service.</Description>
+    <AssemblyTitle>Microsoft Azure Provisioning Service Management</AssemblyTitle>
+    <AssemblyName>Microsoft.Azure.Management.DeviceProvisioningServices</AssemblyName>
+    <VersionPrefix>0.9.0-preview</VersionPrefix>
```
Comment (shahabhijeet) [inline src/SDKs/DeviceProvisioningServices/Management.DeviceProvisioningServices/Microsoft.Azure.Management.DeviceProvisioningServices.csproj]: this is < Version > and not < versionPrefix >

File: src/SDKs/DeviceProvisioningServices/Management.DeviceProvisioningServices/Microsoft.Azure.Management.DeviceProvisioningServices.csproj [hunk 2]
```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
```
Comment (shahabhijeet) [inline src/SDKs/DeviceProvisioningServices/Management.DeviceProvisioningServices/Microsoft.Azure.Management.DeviceProvisioningServices.csproj]: change the type from below this line to above this line

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/IotHub/Management.IotHub/generate.cmd [hunk 3]
```diff
@@ -4,4 +4,4 @@
 ::
 
 @echo off
-call %~dp0..\..\..\..\tools\generate.cmd iothub/resource-manager %*
+call %~dp0..\..\..\..\tools\generate.cmd provisioningservices/resource-manager %*
```
Comment (shahabhijeet) [inline src/SDKs/IotHub/Management.IotHub/generate.cmd]: @montgomp why iotHub has been changed to provisioning services?

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/IotHub/Management.IotHub/generate.cmd (hunk 3 above)
Comment (montgomp) [reply src/SDKs/IotHub/Management.IotHub/generate.cmd]: Thanks for the catch.

File: src/SDKs/DeviceProvisioningServices/Management.DeviceProvisioningServices/Microsoft.Azure.Management.DeviceProvisioningServices.csproj (hunk 2 above)
Comment (montgomp) [reply src/SDKs/DeviceProvisioningServices/Management.DeviceProvisioningServices/Microsoft.Azure.Management.DeviceProvisioningServices.csproj]: Someone should look into the examples as it seems every project has "below" where it should be "above"

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**